GL-js-hw-01/task-04.js

```diff
@@ -0,0 +1,23 @@
+const credits = 23580;
+const pricePerDroid = 3000;
+
+const inputCount = prompt('Колличество дроидов для покупки: ');
+
+if (inputCount === null) {
+  console.log('Отменено пользователем!');
+} else {
```

Comment:
Тут можна змінити трішки умову 
```suggestion
} else  if (isNaN(Number(inputCount)))  { 
 ...
} else {
 ....
}
```